### ISSUE TITLE
Make it available for macos

### ISSUE THREAD

**Issue by ashishbhatiya18** [lawl/NoiseTorch#151]
Would it be possible to port it and include distribute macos binary as well?

**Comment by lawl** [comment]
No, that would require essentially rewriting it from scratch for macos. (The same goes for windows).

So let me ask you a different question. If it was (hypothetically) available on mac today, would you pay for it, and if so, how much?

**Comment by gianpaj** [comment]
10 USD. 1 year license

**Comment by lawl** [comment]
closing this. not enough interest. keep in mind that i would have to acquire apple hardware etc.

**Comment by gianpaj** [comment]
or rent one in the cloud e.g. `mac1.metal` on AWS EC2 starting at $1.083 per hour 
https://aws.amazon.com/ec2/dedicated-hosts/pricing/#Pricing_for_Dedicated_Hosts
> Please note that On-Demand Mac1 Dedicated Hosts have a minimum host allocation and billing duration of 24 hours.

or https://www.macincloud.com/pages/payg.html at 1USD per hour with min 30 USD deposit but I believe you can stop and start on-deman

**Comment by culebrasolida** [comment]
Come on, there's Krisp available for mac like since forever.

**Comment by breuerfelix** [comment]
https://www.utterly.app -> free to use, pretty minimalistic and working as intended. also available via homebrew.  
google the same and found it!

**Comment by linsomniac** [comment]
Patreon and happy noisetorcher here.  Just looking for it on MacOS because I've been so happy with it under Linux.  Sounds like another couple options are available at the same price level for what I've been paying in Patreon, $5/mo.

**Comment by lawl** [comment]
no, that would solve pretty much no problems. mac and windows both require kernel drivers to achieve this. linux is the only OS where we can conveniently do this from userspace.

at least on windows, writing a kernel driver pretty much requires a company to handle EV code signing certs for you because they are simply not handed out to individuals. OSX is probably similar.

These would be a huge time investment, money investment, and involve either founding a company, or find one that would sponsor these things.

In short, OSX or windows are simply not happening without a decent chunk of money involved due to external factors beyond our control. Thus, I would need to sell NoiseTorch for OSX/Windows, while i'm happy to give it away on Linux, because it's something I primarily made for myself, and it's about 100x easier on linux.
